src/main/java/org/kohsuke/github/GHIssueComment.java

```diff
@@ -35,7 +35,7 @@
     GHIssue owner;
 
     private String body, gravatar_id;
-    private GHUser user;
+    private GHUser user; // not fully populated. beware.
```

Comment:
Seems like this sort of comment would better be placed in the Javadoc of the getter.